.github/pull_request_template.md

```diff
@@ -0,0 +1,53 @@
+<!-- Please complete the template below as part of your PR -->
+
+> ⚠️ It's important to note that only internal Microsoft teams and partners with "contributor" rights for this repo will be allowed to submit new built-ins and changes, all other PR submitters will have their PRs automatically rejected.
+
+# Built-in Template Spec Contribution
+
+All PR submitters must read [Authoring and Contributing Built-ins Documentation](../built-ins/authoring-built-ins.md), and must also fill out the rest of this pull request template before submitting their PR.
+
+## What kind of submission is this? 
+
+Check the box that's relevant to the type of PR being submitted:
+
+* [ ] Entirely new built-in Template Spec
+* [ ] New version to an existing Template Spec
+* [ ] An update to an existing version
+
+... and for the relevant section below, provide the requested detail(s):
+
+----
+
+### If it's an entirely new built-in:
+
+**What customer scenario/purpose does this new built-in fulfill?:** 
+
+_- Provide the details here -_
+
+### If it's for a new version in an existing built-in:
+
+**What changes are introduced with this version?:** 
+
+_- Provide the details here -_
+
+### If it's an update to an existing version:
+
+**What changes are being made to the existing version?:** 
+
+_- Provide the details here -_
+
+**Why are these changes being made in an existing version instead of introducing a new version?:** 
+
+_- Provide the details here -_
+
+<br/>
+
+## Submission Checklist
+
+Please confirm that you've completed the following PR requirements:
```

Comment:
Might want to add one along the lines of "I've verified that there is not an existing built-in template spec that accomplishes the same or very similar scenario"